mlir/test/Target/LLVMIR/Import/tune-cpu.ll

```diff
@@ -0,0 +1,9 @@
+; RUN: mlir-translate -import-llvm -split-input-file %s | FileCheck %s
```

Comment:
Should this test also be guarded with `REQUIRES: x86-registered-target`?

Reply:
Thank you so much for the catch!  Yes, this guard is absolutely needed.

Reply:
Hm, ninja `check-mlir` worked for me just fine on Aarch64 🤔 I think that `mlir-translate` doesn't really care :) Put differently, `REQUIRES` is not needed here.

Reply:
Interesting!

Did you build your LLVM with the x86 target enabled (besides Aarch64)? If not, `mlir-translate` may indeed be fine with the attribute independent of the target architecture. Note that `mlir-translate` runs `llvm::verifyModule` which I thought could be unhappy with unsupported target specific attributes.

Reply:
I ran into failures if I didn't include the guard.  I have separate builds for x86 and Aarch64.

Reply:
I have X86 disabled - I just double checked. And yes, this test works for me just fine without the guard. Alexis, what error do you get when it fails for you? 

To me, it would make sense if `mlir-translate` didn't care - it doesn't have to interpret these attributes, does it? 

Btw @AlexisPerry , there's no need to clone "tune-cpu.f90" as [tune-cpu-llvm-aarch.f90](https://github.com/llvm/llvm-project/pull/96688/files#diff-879569867a123346813864ce36bc15249acb8b57bc335c808eb4d30818ec8685). Instead, you can use conditional RUN lines like here: 
* https://github.com/llvm/llvm-project/blob/main/flang/test/Driver/pic-flags.f90

Reply:
I have been trying to reproduce my error ever since saying I had one and have been unsuccessful.  It now appears that mlir-translate doesn't care about the target.

@banach-space  That is a much better way of doing things.  Thank you for the suggestion and I will make the changes.

Reply:
> To me, it would make sense if mlir-translate didn't care - it doesn't have to interpret these attributes, does it?

It doesn't do anything with them except for running the LLVM verifier. If things pass on Aarch64, then the guards are not necessary.